lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -37,6 +37,10 @@ html.video-fullscreen{
       color: $white;
     }
   }
+
+  @media print {
```

Comment:
I see printing as being useful from two perspectives: creating PDF versions of a page for students to download, and allowing students to print. If I'm making a PDF version, I don't particularly want "Staff Info Only" plastered all over it. I'd much prefer to create something that is as close to what students will see as possible.